bundles/org.openhab.binding.icloud/src/main/java/org/openhab/binding/icloud/internal/RetryException.java

```diff
@@ -23,7 +23,7 @@
 @NonNullByDefault
 public class RetryException extends RuntimeException {
 
-    private static final long serialVersionUID = 1L;
+    private static final long serialVersionUID = 4367157567009973413L;
```

Comment:
I prefer 1L here. Putting a serialVersionUID is best practice to allow "manual" management of the UID if the class is serialized and we want to maintain backward compatibility. From my point of view this is more explicit and easier to handle if it is a simple value (like 1) and not the one which would have been generated by the javac.

Reply:
Since this is an exception which is unlikely to be persisted anywhere between versions, I agree the value is not important. I'm sorry for the confusion.

Reply:
@maihacke @jlaur both is fine by me. At the moment it is 1L due to this change being reverted.